docs/labs/index.rst

```diff
@@ -18,9 +18,9 @@ If we get lots of positive feedback about any one of these projects then we will
 Flocker Docker plugin with CLI and GUI
 ======================================
 
-By way of example, here is a 55 second demo of the :ref:`Flocker Docker plugin <labs-docker-plugin>` provisioning portable Flocker volumes and moving them between hosts directly from the Docker CLI.
+By way of example, here is a 55 second demo of the :ref:`Flocker Docker plugin <docker-plugin>` provisioning portable Flocker volumes and moving them between hosts directly from the Docker CLI.
```

Comment:
For consistency, I think references on this page should be to the "Flocker Plugin for Docker" rather than "Flocker Docker plugin"